path_planning:
  ros__parameters:
    frequency: 30
    global_grid_topic: /global_map
    odom_topic: /odom
    goal_topic: /goal_pose
    path_topic: /path
    robot_collision_radius: 3.
    goal_radius: 1.
    steering_value: 0.3
    path_discrete: 0.5
    robot_base_frame: base_footprint
    grid_resolution: 0.1